apps/web/src/app/api/mint-nft/client-utils.ts

```diff
@@ -93,35 +93,40 @@ export async function sendTransaction(params: {
         // console.log(`jobResult api call result:`, answer);
 
         // Check for various error conditions
-        if (answer.jobStatus === "failed") {
+        if (answer.jobStatus === "failed" || answer.error) {
```

Comment:
we should first confirm the answer structure. If error is empty string, then its fine